setup.py

```diff
@@ -31,8 +31,8 @@ def long_description():
         'Bug Reports': 'https://github.com/dopplerhq/python-doppler-env/issues',
         'Source': 'https://github.com/dopplerhq/python-doppler-env',
     },
-    author='Doppler',
-    author_email='support@doppler.com',
+    author='Ryan Blunden',
+    author_email='ryan.blunden@doppler.com',
```

Comment:
Should this change? If so, what is the reasoning?

Reply:
Only changed to be consistent with the gitops-secrets repo. I'm happy to leave as is if we think that's better.

Reply:
What do you think is better, not sure what the pros & cons are 🤔